consensus/obcpbft/pbft-core_test.go

```diff
@@ -793,6 +793,61 @@ func TestSendQueueThrottling(t *testing.T) {
 	}
 }
 
+// Test for issue #1091
+// Once the primary ran out of sequence numbers, it would queue requests into a map, and resubmit them in arbitrary order
+// This is incorrect, they need to be resubmitted in the order of their timestamps
```

Comment:
Incorrect is perhaps a strong word, but the arbitrary ordering resulted in rather undesirable in behavior in the context of batch, which expects for requests to be ordered by the PBFT primary in the same order in which it submitted them to the PBFT primary.